src/rsz/include/rsz/Resizer.hh

```diff
@@ -404,6 +404,16 @@ class Resizer : public dbStaState
   void journalBeginTest();
   void journalRestoreTest();
 
+  void openSpefFile(const char* path);
+  void closeSpefFile();
+  void writeSpefHeader(Corner* corner, dbNetwork* network);
+  void writeSpefPorts(Corner* corner, dbNetwork* network);
+  void writeSpefNet(Corner* corner,
+                    const Net* net,
+                    Parasitic* parasitic,
+                    dbNetwork* network,
+                    Parasitics* parasitics);
```

Comment:
Can the writer be split into its own class?  I don't see a dependency on the rest of resizer in this code and it is shared with grt.